app/assets/stylesheets/text.scss

```diff
@@ -21,3 +21,8 @@ hr {
   box-shadow: 0px 2px #E0E0E0;
   margin: 7.5% 0 5% 0;
 }
+
+.thin {
```

Comment:
Reading this class name - i'm not sure what `.thin` is making thin. It would be good to add a bit more context to it for example `.section-break__thin` or however we've identified other things like that

Reply:
Renamed to `section-break-thin`.
Seems to fit in with how we've named other classes.